.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "libraries/Adafruit-PWM-Servo-Driver-Library"]
 	path = libraries/Adafruit-PWM-Servo-Driver-Library
-	url = https://github.com/adafruit/Adafruit-PWM-Servo-Driver-Library/
+	url = http://github.com/adafruit/Adafruit-PWM-Servo-Driver-Library/
```

Comment:
For some reason CircleCI doesn't like submodules with `https` domains, this was a nice-ish fix https://circleci.com/gh/sourcebots/servo-firmware/5